scss/6-components/_targeted-content.scss

```diff
@@ -10,16 +10,15 @@
     display: block;
     color: $cads-language__brand-primary;
     background: $cads-palette__white;
-    border: solid $cads-border-width-small $cads-language__brand-primary;
+    border: 1px solid $cads-language__link-colour;
```

Comment:
is there a reason that you aren't using `$cads-border-width-small` here anymore?

Reply:
Small is 2px, but the designs specify 1px. But maybe we need a variable for that.

Reply:
It's 1px - I just checked 😄 
https://github.com/citizensadvice/design-system/blob/35256977409c5e1998757702809561805f96ade7/scss/1-settings/_borders.scss#L2

Reply:
😆 Whoops!